js/script.js

```diff
@@ -1,61 +1,122 @@
+    var numbersOfRounds = prompt('Ile rund?');  
+    if (isNaN(numbersOfRounds) || numbersOfRounds <= 0) {
+            alert('Podałeś nieprawidłową wartość licznik ustawiony na 5 rund');
+            return numbersOfRounds = 5;       
+        } else {
+                return numbersOfRounds;
+        }       
+};
+   
+function registerListeners() {
+    beginButton.addEventListener('click', function() {
+        displayNone()
+        var totalRoundsForGameAsString = askHowManyRounds();
+        totalRoundsForGame = Number(totalRoundsForGameAsString);
+        showNumberOfRounds.innerHTML = 'ilość rund: ' + totalRoundsForGame;
+        display();       
+    });
+    stone.addEventListener('click', function() {
+        choseAnItem('kamień');        
+    });
+    paper.addEventListener('click', function() {
+        choseAnItem('papier');     
+    });
+    scissors.addEventListener('click', function() {
+        choseAnItem('nożyczki');          
+    });      
+};
 
+function getRandomChoice() {
+    var randomNumber = (Math.floor(Math.random()*3));
+    var choices = ['kamień', 'papier', 'nożyczki'];
+    return choices[randomNumber];
+};
 
+function showResult(playerChoice, computerChoice) {
+    if ((playerChoice == 'kamień' && computerChoice == 'nożyczki') || (playerChoice == 'papier' && computerChoice == 'kamień') || (playerChoice == 'nożyczki' && computerChoice == 'papier')) {
+        showWhoWon.innerHTML = 'Wygrana!' + '<br/>' + 'Twój wybór: ' + playerChoice + '<br/>' + 'Wybór komputera: ' + computerChoice + '<br>';     
+        wins = wins + 1 ,winner.innerHTML = wins;
+    } else if ((playerChoice == 'kamień' && computerChoice == 'kamień') || (playerChoice == 'papier' && computerChoice == 'papier') || (playerChoice == 'nożyczki' && computerChoice == 'nożyczki')) {       
+        showWhoWon.innerHTML = 'Remis!' + '<br/>' + 'Twój wybór: ' + playerChoice + '<br/>' + 'Wybór komputera: ' + computerChoice + '<br>';       
+        draws = draws + 1, remixed.innerHTML = draws;
+    } else {      
+        showWhoWon.innerHTML = 'Porażka!' + '<br/>' + 'Twój wybór: ' + playerChoice + '<br/>' + 'Wybór komputera: ' + computerChoice + '<br>';       
+        loss = loss + 1, losser.innerHTML = loss;
+    }
+        game = game + 1, games.innerHTML = game;
+        endGame(totalRoundsForGame,  game);  
+};
 
-function startRounds() {
-    
-}
+function choseAnItem(playerChoice) {
+    var computerChoice = getRandomChoice(); 
+    showResults.innerHTML = 'Gracz:' +  ' Komputer: ' + '</br>' + playerChoice + ' ' + computerChoice;
+    beginButton.style.display = "none";   
+    showResult(playerChoice, computerChoice);  
+};
 
-function showButtons() {
+function endGame(totalRoundsForGame,  game) {
+    if (totalRoundsForGame ===  game) {
+        endOfGame.innerHTML = 'Abt rozpoczać od nowa kliknij nowa gra';
+        stone.style.display = "none";
+        paper.style.display = "none";
+        scissors.style.display = "none";
```

Comment:
masz funkcje od pokazywania przycisków, to albo zrób analogiczną od ukrywania.. albo najlepiej zrób z niej jedną, bardziej konfigurowalną funkcję, która przyjmie parametr "none" lub "block' i wtedy odpowiednio się zachowa